### PR TITLE
Draft: Single test refactor

### PR DESCRIPTION
**Pull request by Alrobbertz**

### CODE REVIEW

**Comment by Alrobbertz** [comment]
Added Ignores for E203 in the `setup.cfg` bases on [this issue](https://github.com/psf/black/issues/315) and the black guidance [here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices)

**Comment by dbarrous** [comment]
My bad I accidentally opened this from Draft

File: pyproject.toml [hunk 1]
```diff
@@ -26,10 +26,9 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
 ]
 dependencies = [
-  'astropy>=4.1.0',
-  'numpy>=1.16.0',
   'hermes_core @ git+https://github.com/HERMES-SOC/hermes_core/',
-  'ccsdspy @ git+https://github.com/ddasilva/ccsdspy.git'
+  'ccsdspy @ git+https://github.com/ddasilva/ccsdspy.git',
+  'ndcube>=2.1.3', # Temporary while not part of hermes_core
```
Comment (Alrobbertz) [inline pyproject.toml:31]: Temporarily aded this line while https://github.com/HERMES-SOC/hermes_core/pull/93 is open. Once the PR in Core completes, ndcube will be installed with the `hermes_core` dependency.

**Comment by ehsteve** [comment]
This should be closed right @Alrobbertz?

**Pull request closed without merging**